plugins/available/percol.plguin.bash

```diff
@@ -0,0 +1,13 @@
+# Install
```

Comment:
There's a typo in the file name. Please change `plguin` to `plugin` in the name.